Description: Remove coveralls
 Remove the unnecessary coveralls dependency
Author: Balasankar C <balasankarc@autistici.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/helper.rb
+++ b/tests/helper.rb
@@ -1,14 +1,14 @@
 require 'excon'
 
-if ENV['COVERAGE']
-  require 'coveralls'
-  require 'simplecov'
-
-  SimpleCov.start do
-    add_filter '/spec/'
-    add_filter '/test/'
-  end
-end
+#if ENV['COVERAGE']
+  #require 'coveralls'
+  #require 'simplecov'
+
+  #SimpleCov.start do
+    #add_filter '/spec/'
+    #add_filter '/test/'
+  #end
+#end
 
 require File.expand_path(File.join(File.dirname(__FILE__), '../lib/fog/atmos'))
 
@@ -33,4 +33,4 @@
 
 def array_differences(array_a, array_b)
   (array_a - array_b) | (array_b - array_a)
-end
\ No newline at end of file
+end
